toolchain/check/eval.cpp

```diff
@@ -766,10 +766,6 @@ static auto ReplaceAllFieldsWithConstantValues(EvalContext& eval_context,
     -> bool {
   auto arg0 =
       GetConstantValueForArg(eval_context, inst->arg0_and_kind(), phase);
-  if (!IsConstant(*phase)) {
```

Comment:
It seems like we'd still produce error phase when combining error + runtime if we take one of the other paths that bail out on `<error>`. Also, shouldn't we still bail out here if `*phase` is runtime?

Perhaps we could instead change `IsConstant` itself to return `true` for the error phase, so that it agrees with our general model that the error value is a constant?